UserInterface/CardViewDemo/CardViewDemo/CardViewDemo/ControlTemplatePage.xaml

```diff
@@ -0,0 +1,75 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:d="http://xamarin.com/schemas/2014/forms/design"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:controls="clr-namespace:CardViewDemo.Controls"
+             mc:Ignorable="d"
+             x:Class="CardViewDemo.ControlTemplatePage"
+             Title="ControlTemplate XAML Demo"
+             Padding="10">
+    <ContentPage.Resources>
+        <ResourceDictionary>
+            <ControlTemplate x:Key="CardViewCompressed">
+                <Grid>
+                    <Grid.RowDefinitions>
+                        <RowDefinition Height="100" />
+                    </Grid.RowDefinitions>
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="100" />
+                        <ColumnDefinition Width="100*" />
```

Comment:
Does the asterisk after the number have any effect? If you just want the second column to take the remaining space just use *.

Reply:
This came from a misunderstanding of the asterisk and worked for the wrong reasons. Fixed.